src/main/java/com/googlecode/jsonrpc4j/JsonRpcBasicServer.java

```diff
@@ -460,12 +490,34 @@ protected Object getHandler(String serviceName) {
 	 */
 	private JsonNode invoke(Object target, Method method, List<JsonNode> params) throws IOException, IllegalAccessException, InvocationTargetException {
 		logger.debug("Invoking method: {} with args {}", method.getName(), params);
-		Object[] convertedParams = convertJsonToParameters(method, params);
-		if (convertedParameterTransformer != null) {
```

Comment:
In whole code used style like
`if (a == b) {`
Marked lines matched this code style in sources. Please check again last commit.